include/DDpackage.h

```diff
@@ -196,7 +197,7 @@ namespace dd {
         Edge         add2(const Edge& x, const Edge& y);
         Edge         multiply2(const Edge& x, const Edge& y, unsigned short var);
         ComplexValue innerProduct(const Edge& x, const Edge& y, int var);
-        Edge         trace(const Edge& a, const std::bitset<MAXN>& eliminate, unsigned short alreadyEliminated = 0);
+        Edge         trace(const Edge& a, const std::vector<bool>& eliminate, unsigned short alreadyEliminated = 0);
```

Comment:
AFAIK std::vector<bool> should be avoided as much as possible. (https://isocpp.org/blog/2012/11/on-vectorbool)

Reply:
I think it works well for our usage and is preferable to including boost's dynamic_bitset

Reply:
it is definitely preferable to including boost. I guess we can keep `std::vector<bool>`